### ISSUE TITLE
Fix inconsistent social icons in the footer

### ISSUE THREAD

**Issue by Aloso** [rust-lang/beta.rust-lang.org#558]
# The icons in the footer need to be changed.

Current situation:

![rust social icons](https://user-images.githubusercontent.com/15658558/49540608-c8c46400-f8d0-11e8-987d-8849c5e5cf37.png)

My proposal:

![rust social icons 2](https://user-images.githubusercontent.com/15658558/49541972-89981200-f8d4-11e8-97d8-e3f8076a7f3f.png)

## Reasons

- The icons look **inconsistent**, since the YouTube icon is the only flat icon.
- The GitHub and Discord icons are **outdated**
- The GitHub and Discord icons are **hard to recognize** due to the significant differences from the original.
- The icons **do not follow the guidelines** by GitHub, Twitter and Discord. Each of these companies has established rules for how their trademarks may be used and modified.

## Official icons/trademarks

![github icons](https://user-images.githubusercontent.com/15658558/49540902-83546680-f8d1-11e8-9e42-02470b073f8c.png)
![discord official logo](https://user-images.githubusercontent.com/15658558/49540906-86e7ed80-f8d1-11e8-942d-d1d2ea12f1b1.png)
![twitter logo](https://user-images.githubusercontent.com/15658558/49540958-aa129d00-f8d1-11e8-8fc0-15e72a3694ac.png)
![youtube icons](https://user-images.githubusercontent.com/15658558/49542356-a84ad880-f8d5-11e8-9bd3-b9da25f4c2c1.png)

PS: A tooltip (short text displayed when the mouse is over the icon) would be nice. It's really easy:

```html
<img src="/static/images/youtube.svg" alt="youtube logo" title="YouTube">
```

**Comment by ashleygwilliams** [comment]
i agree! i would welcome a PR for this :)

**Comment by The-lady-developer** [comment]
Hi @ashleygwilliams! I want to work on this.

**Comment by ashleygwilliams** [comment]
@The-lady-developer that's awesome! let me know if you have any questions :)

**Comment by ashleygwilliams** [comment]
closed by #569 - sorry @The-lady-developer - if there's another issue you'd like to take on, let me know!
